<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleEditMenu</class>
 <widget class="QMainWindow" name="SimpleEditMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>简单调查:编辑菜单</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentPositionLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>54</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>下一个位置:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentLineLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentLineDataLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentMarkLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark:</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentMarkDataLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastLineLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastLineDataLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastMarkLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>290</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastMarkDataLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>290</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="deleteLineLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Line:</string>
    </property>
   </widget>
   <widget class="QLabel" name="deleteLineDataLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>350</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lastPositionLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>回到最后位置:</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除行:</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="returnLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>返回调查</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
